wp-content/themes/core/components/statistic/Statistic.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Tribe\Project\Templates\Components;
+
+use Tribe\Project\Components\Component;
+
+/**
+ * Class Statistic
+ *
+ * @property string   $tag
+ * @property string[] $classes
+ * @property string[] $attrs
+ * @property string[] $container_classes
+ * @property array    $value
+ * @property array    $label
+ */
+class Statistic extends Component {
```

Comment:
Interesting to create a "statistic" component versus just being an element of the stats block. Curious if you foresee using this outside the context of the block or another reason for a separate component?

Reply:
Initially I had looked at a few component libraries and saw others included it as its own component. At the time I thought it could potentially be designed to be used in a card header, maybe a spin off of media + text where stat replaces media. About halfway through building it, I started second guessing if it really needed to be separate. Since I was less involved in discussions on what components should be, I'm open to your opinion and @ryanurban's on simplifying to just a part of the Stats block.

Reply:
Let's start with it as a component. Does give us a lot of additional flexibility.